lingpy3/algorithm/cython/_calign.py

```diff
@@ -165,7 +165,7 @@ def globalign(
             almB += [seqB[i-1]]
             i -= 1
             j -= 1
-        else:
+        else:  # pragma: no cover
```

Comment:
okay, it's more difficult to trigger all of those cases, but at least I made sure that there is no error. I started adding more tests to test_cython, but it is tedious, yet I will try to reach more than 95% at some point (yet not today).